app/helpers/bets_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BetsHelper
+  def creator?
+    current_user == @bet.creator
+  end
+
+  def resolved?
```

Comment:
1) Name is confusing, since the field is called `active`, this should probably be `active?` which is already provided by default by Rails/ActiveRecord for boolean fields, removing the need for this helper as you can directly used `active?`

2) This would rely on `@bet` being defined somewhere else as an instance variable, which is confusing and unreliable, so instead a bet should be provided as a parameter for scenarios like this